profiles/prod.config.js

```diff
@@ -5,5 +5,6 @@ export default Object.assign({}, base, {
   dest: 'dist/umd/esri-loader.min.js',
   plugins: [
     uglify()
-  ]
+  ],
+  context: 'window'
```

Comment:
We definitely don't need this one. It would get merged in from the base (if we end up keeping it there).

Reply:
Good point. I removed that entry from `prod.config.js` since they are merged.